src/main/java/be/seeseemelk/mockbukkit/scheduler/BukkitSchedulerMock.java

```diff
@@ -1,15 +1,14 @@
 package be.seeseemelk.mockbukkit.scheduler;
 
+import java.util.ArrayList;
 import java.util.LinkedList;
 import java.util.List;
-import java.util.concurrent.Callable;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.Executors;
-import java.util.concurrent.Future;
+import java.util.concurrent.*;
```

Comment:
Please avoid the usage of wildcard imports at all times.

Reply:
Oh, didn't notice those auto-optimizations of my IDE, will fix this